src/OpenTelemetry.Api/Baggage.cs

```diff
@@ -46,21 +46,34 @@ internal Baggage(Dictionary<string, string> baggage)
         /// <summary>
         /// Gets or sets the current <see cref="Baggage"/>.
         /// </summary>
+        /// <remarks>
+        /// Note: <see cref="Current"/> returns a forked version of the current
+        /// Baggage. Changes to the forked version will not automatically be
+        /// reflected back on <see cref="Current"/>. To update <see
+        /// cref="Current"/> either set <see cref="Current"/> to a new instance
+        /// or use one of the static methods that target <see cref="Current"/>
+        /// as the default source. Examples:
+        /// <code>
```

Comment:
can you reverse the order of the example codes below, so that it matches the Note: section.

Reply:
I reversed your request to reverse the code examples and instead reversed the notes. I wanted to have the preferred approach (static methods) first. I also updated the README to the safer static API.

Reply:
> I reversed your request to reverse the code examples and instead reversed the notes.

This makes my day 😵😆

Reply:
Ya had some fun with that one 😄 Also very excited about the pun opportunities for my next PR: "Lost Baggage"